docker/manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "maintainer": "help@datadoghq.com",
   "manifest_version": "0.1.0",
-  "max_agent_version": "6.0.0",
+  "max_agent_version": "5.14",
```

Comment:
As we're probably going for a 5.14.1 release, I'd set the max_version to 5.14.1, and include this PR in the 5.14.1 release